game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/renderer/JsonRenderer.scala

```diff
@@ -49,7 +49,8 @@ class JsonRenderer extends BaseMapRenderer {
                     ("x" -> b.getPosition().getBlockNumber())
                   ) ~
                   ("surfaceObject" -> b.getMapObject()) ~
-                  ("occupiedByPlayerId" -> b.getOccupiedByPlayerWithId())
+                  ("occupiedByPlayerId" -> b.getOccupiedByPlayerWithId()) ~
```

Comment:
What is the ~ sign at the end for?

Reply:
it's part of the library we use to map types to json objects, basically a property separator in this case